package.json

```diff
@@ -9,6 +9,7 @@
     "apollo-client-preset": "^1.0.8",
     "apollo-link-http": "^1.5.4",
     "autosuggest-highlight": "^3.1.1",
+    "date-fns": "^1.29.0",
```

Comment:
Date-fns is much lighter than moment since it deals with date objects directly. Let's use it where performance matters.